pkg/vpcmodel/explainabilityConnectivity.go

```diff
@@ -88,23 +89,29 @@ func (configsMap MultipleVPCConfigs) ExplainConnectivity(src, dst string, connQu
 	if vpcConfig == nil {
 		// No error and also no matching vpc config for both src and dst: missing cross-vpc router.
 		// No VPCConfig to work with in this case, thus, this case is treated separately
-		return &Explanation{nil, connQuery, nil, src, dst, nil, nil, false, nil}, nil
+		return &Explanation{nil, connQuery, nil, src, dst,
+			nil, nil, false, nil}, nil
```

Comment:
this can be shortened as well (no need  to assign default values, only explicitly assign relevant fields)

Reply:
right. done.

Reply:
pushed now